SalesforceCommerceCloud/pwa-kit
# [V2] 3PP bump Verdaccio to latest

GUS 3PP request: https://gus.lightning.force.com/lightning/r/ADM_Third_Party_Software__c/a0qEE0000002RSLYA2/view



# Description



Security rejected the 3PP verdaccio@4.12.2 because it's deprecated. See Chatter section on the [GUS 3PP request](https://gus.lightning.force.com/lightning/r/ADM_Third_Party_Software__c/a0qEE0000002RSLYA2/view).

The PR updates Verdaccio o latest v5.9.0.

The breaking changes detailed in the [v5 migration guide](https://verdaccio.org/blog/2021/04/14/verdaccio-5-migration-guide/) do not affect us. We can safely bump Verdaccion to the latest version.

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Bump Verdaccio to latest.

# How to Test-Drive This PR

1. Run the create-mobify-dev.js script:
```
GENERATOR_PRESET=test-project node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js --outputDir /tmp/generated-project
```
2. After successfully generating a project:
  - Check the Verdaccio logs for any errors. Verdaccio logs file: `packages/pwa-kit-create-app/local-npm-repo/verdaccio.log`
  - Check the generated projects work as expected.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
